charity_finder/views.py

```diff
@@ -24,8 +25,12 @@ def get_map():
 
     # Filtering by remaining funding money by Region
     # TODO:  Include more regions
-    GOAL_LIMIT = 100_000
-    REGION_NAME = "Africa"
+    GOAL_LIMIT: Final[
+        int
+    ] = 100_000  # constant marked final, can't assign another value to it
+    REGION_NAME: Final[str] = "Africa"
```

Comment:
nice, I did a YT video on this recently :)

Reply:
Awesome! I will check it out. This was so great to learn.

Reply:
https://www.youtube.com/watch?v=XjQi8v9OPzw > funny you did it better than me by also typing the Final (see comment on video)